FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    a more broad pen can be observed on <locus target="#202v"/>.</note>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <ab type="script"/>
+                                <locus from="171r" to="174v"/>
+                                <seg type="script">Angular, but less careful, taller and more hasty than <ref target="#h1" /> with characters  
+                                    alternately tilted to the right or left.</seg>
+                                <seg type="ink">Black.</seg>
+                                <date from="1400" to="1500" cert="medium" datingMethod="paleography"/>
+                            </handNote>
+                            <handNote xml:id="h3" script="Ethiopic">
+                                <ab type="script"/>
+                                <locus from="208r" to="209v"/>
+                                <seg type="script">Careful and angular, smaller and more careful than the previous hands.</seg>
+                                <seg type="ink">Black.</seg>
+                                <date from="1400" to="1500" cert="low" datingMethod="paleography"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote type="drawing" xml:id="d1">
+                                <locus target="#211v"/>
+                                <desc>Rough ink drawing depicting a bird or another animal.</desc>
+                            </decoNote>
+                            <decoNote type="ornamentation" xml:id="d2">
+                                <locus target="#207v"/>
+                                <date from="1400" to="1600" cert="low" datingMethod="decoration"/>
+                                <desc>Four small ornaments marking the end of a text.</desc>
+                            </decoNote>
+                            <decoNote type="frame" xml:id="d3">
+                                <locus target="#212v"/>
+                                <desc>Improvised frame for a small text on the upper part of the page, that was erased.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="210r" to="210v"/>
+                                    <desc type="Unclear">Text erased from protective leaf, not legible except for individual letters or single words.</desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="210r" to="210v"/>
+                                    <desc type="Unclear">Text erased from protective leaf, partly not legible: <foreign xml:lang="gez">ልብሳ፡ ማሓፈ፡ ፩፩
```

Comment:
If this is the text of the addition, it can be included in `<q>`

Reply:
Do you mean, it need a <q> and </q> before and after the quotation inside the addition-record?

Reply:
N.B.: if writing <> on here, it is necessary to mark these up as code, or their content will not be printed:
![image](https://user-images.githubusercontent.com/19650382/228194499-b83c7ec6-d830-46fe-99eb-c757463be295.png)

Reply:
According to the guidelines, it should be outside of `<desc>` (I don't know how much of a hard rule that is). This also makes `<foreign>` unnecessary:
```
 <desc type="Unclear">Text erased from protective leaf, partly not legible.</desc>
<q xml:lang="gez">ልብሳ፡ ማሓፈ፡ ፩፩
                                        <gap reason="illegible"/>ምግዕ፡ ዘዕፅ፡ ወ<gap reason="illegible"/>ቀፉ፡ ፪ <gap reason="illegible"/>ዮ ር <gap reason="illegible"/>
                                        ተ፡ ጠር <gap reason="illegible"/> ንዱ <gap reason="illegible"/> ሀሎ፡ ውስተ፡ ቤተ፡ <gap reason="illegible"/></q>
```

Reply:
Alright! Done.